src/App.tsx

```diff
@@ -1,40 +1,65 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
-
-export class App extends React.Component<{}, State> {
-  state: State = {};
-
-  render() {
-    return (
-      <div className="page">
-        <div className="page-content">
-          <div className="box">
-            <div className="field">
-              <label htmlFor="search-query" className="label">
-                Search movie
-              </label>
-
-              <div className="control">
-                <input
-                  type="text"
-                  id="search-query"
-                  className="input"
-                  placeholder="Type search word"
-                />
-              </div>
+export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const movieSearch = (event: any) => {
```

Comment:
```suggestion
  const movieSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
```
Тип `any` лучше вообще никогда не использовать